.github/workflows/build.yaml

```diff
@@ -353,22 +353,22 @@ jobs:
 
       - name: Build main images
         run: |
-          GOOS=linux GOARCH=${{ matrix.arch }} make docker-build-main-image
+          GOOS=linux GOARCH=${{ matrix.arch }} scripts/lib.sh retry 5 true make docker-build-main-image
```

Comment:
Does this actually push? If not, retrying the build itself is not necessary...?

Reply:
> Does this actually push? If not, retrying the build itself is not necessary...?

The flake in the build is in the `FROM <image>` part, so its the pull that I'm looking to retry. But you do bring up a good point. If someone breaks the dockerfile it'll retry 5 times with backoff before they get an error :scream: And at about ~3 minutes per build, that would be suboptimal. I'll drop to 3 retries.